### ISSUE TITLE
A potential bug

### ISSUE THREAD

**Issue by tomato18463** [mpc001/auto_avsr#18]
Hi

I used part of your code in my work, and I find a potential bug (I have not run your original code though). Please can you give it a check? Specifically, [this line](https://github.com/mpc001/auto_avsr/blob/d693142e3ee7130da823b00276af2c97cf7a9d0e/datamodule/av_dataset.py#L14) pads the audio data if its length is smaller than 640 times the corresponding video data length. And [this line](https://github.com/mpc001/auto_avsr/blob/d693142e3ee7130da823b00276af2c97cf7a9d0e/datamodule/av_dataset.py#L32) says the variable `data` has a size of Tx1, so the `torch.nn.functional.pad` function in [this line](https://github.com/mpc001/auto_avsr/blob/d693142e3ee7130da823b00276af2c97cf7a9d0e/datamodule/av_dataset.py#L14C16-L14C39) will result a output size of Tx(1+padding). This seems incorrect to me. I think the padding result is supposed to be (T+padding)x1, and [this line](https://github.com/mpc001/auto_avsr/blob/d693142e3ee7130da823b00276af2c97cf7a9d0e/datamodule/av_dataset.py#L14C16-L14C39) may need to be changed to something like `torch.nn.functional.pad(data, (0, 0, 0, padding), "constant")`. I know I may be wrong as I have not run your original code. Please can you check it anyway?

Thanks!

**Comment by mpc001** [comment]
Hi @tomato18463, I have fixed the bug. Thank you for pointing out this! #20
